.github/workflows/openssl.yml

```diff
@@ -67,4 +67,4 @@ jobs:
       - name: Check LibSSL version
         run: bin/crystal eval 'require "openssl"; p! LibSSL::OPENSSL_VERSION, LibSSL::LIBRESSL_VERSION'
```

Comment:
Shouldn't this use `crystal` instead of `bin/crystal` as well?

ditto elsewhere

Reply:
Oh. I have no idea where those removals of `bin/` come from. Their unintended.